src/Package/Impl/Packages/R/RPackage.cs

```diff
@@ -112,23 +112,27 @@ internal class RPackage : BasePackage<RLanguageService>, IRPackage {
             VsWpfOverrides.Apply();
             CranMirrorList.Download();
 
+            base.Initialize();
+
+            ProjectIconProvider.LoadProjectImages();
+            LogCleanup.DeleteLogsAsync(DiagnosticLogs.DaysToRetain);
+
+            IdleTimeAction.Create(CompleteInit, 20, this.GetType(), VsAppShell.Current);
```

Comment:
(1) is basically add 

RPackage.LoadSettings

and call it from the project adter loading the package